### ISSUE TITLE
Fetch all errors during NSIS compile

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [bleachbit/wishlist#69]
It seems not all errors getting fetched during NSIS compile!
There were some language strings missing, this creates at least a warning, and it seems this didn't get recognized.

**Comment by az0** [comment]
Are you refering to these lines in the log?
>warning 9100: Generating version information for language "0000-Neutral" without standard key "FileDescription"
warning 9100: Generating version information for language "1033-English" without standard key "FileVersion"
warning 9100: Generating version information for language "1033-English" without standard key "LegalCopyright"

**Comment by Tobias-B-Besemer** [comment]
No, not those, but those we should fix also, if we don check in the new installers... but as I think you will land the PR soon... but I can fix it (make a PR) if you like...

**Comment by Tobias-B-Besemer** [comment]
Sorry, didn't made a SS of those I meant, just fixed it... but let me have a look on the old AppVoyer logs...

**Comment by Tobias-B-Besemer** [comment]
Seems warnings are ATM not show... but I make a small PR that they get shown! Then you can try to fetch them...

**Comment by Tobias-B-Besemer** [comment]
Compiling looks IMHO better with https://github.com/bleachbit/bleachbit/pull/693...

![2019-11-12 07_33_50-bleachbit 3 0 1 1372 - AppVeyor](https://user-images.githubusercontent.com/11881073/68647966-7c8d9500-051f-11ea-85d6-25e7d9fea94b.jpg)

![2019-11-12 07_35_26-bleachbit 3 0 1 1372 - AppVeyor](https://user-images.githubusercontent.com/11881073/68647972-81524900-051f-11ea-8d74-f78e5505c741.jpg)

...there are still warnings/errors, but I can dig later into them...
